hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/SplitWALProcedure.java

```diff
@@ -192,6 +189,21 @@ protected void afterReplay(MasterProcedureEnv env){
     if(worker != null){
       env.getMasterServices().getSplitWALManager().addUsedSplitWALWorker(worker);
     }
+  }
+
+  @Override protected void toStringClassDetails(StringBuilder builder) {
+    builder.append(getProcName());
+    if (this.worker != null) {
+      builder.append(", splitWorker=");
+      builder.append(this.worker);
+    }
+    if (this.retryCounter != null) {
+      builder.append(", retry=");
+      builder.append(this.retryCounter);
+    }
+  }
 
+  @Override public String getProcName() {
+    return getClass().getSimpleName() + " " + WAL.stripWALsDirPrefix(getWAL());
```

Comment:
Why only prefix? How about use the WAL directly?

Reply:
You idea is better. Let me do it.

Reply:
hmm... I only have a string... but let me just output the file name instead.